src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -287,8 +287,10 @@ class MessageHelper {
    * @brief Sends button subscription notifications for all buttons
    * that application is subscribed on
    */
-  static void SendAllOnButtonSubscriptionNotificationsForApp(
-      ApplicationConstSharedPtr app, ApplicationManager& app_mngr);
+  static void SendOnButtonSubscriptionNotificationsForApp(
```

Comment:
@mked-luxoft please do not create in message Helpter functions like `Send*`. 
Create smartobject and send it from redumptionDataProcessor

Reply:
@LuxoftAKutsan fixed in83a9f4076feb8eb96f23cccb302c544fa72dcd4b

Reply:
@LuxoftAKutsan fixed in 83a9f4076feb8eb96f23cccb302c544fa72dcd4b